apps/knowledge-base/README.md

```diff
@@ -0,0 +1,92 @@
+# Contentful Knowledge base app
+
+> A Contentful Marketplace app to create new knowledge base websites within a few minutes.
+
+## Features
+
+- Content type creation
+- Sample content creation
+- Content preview
+- Netlify build triggers
+- Website template deploy
+
+## Easiest way to get started
+
+1. Create an account on [contentful.com](https://www.contentful.com/get-started/)
+1. Install the [Knowledge base app](https://www.contentful.com/marketplace/) from Contentful Marketplace
+1. Done!
+
+## Making changes locally
+
+1. [Create a new OAuth application](https://app.netlify.com/user/applications#oauth-applications) on your Netlify account
+1. The `.env.example` file contains the environment variables the project needs
+1. Create the `.env.development` and `.env.production` files, one for each environment respectively, and fill up the environment variables
+1. Note: The env variable `NETLIFY_OAUTH_APP_REDIRECT_URI` must end with `/auth`
+1. [Add the `Knowledge base` app to a Contentful organization](#adding-the-app-to-your-organization)
+1. Install the dependencies by running `npm ci`
+1. `npm start`
+
+### Adding the app to your organization
+
+You need to add the app into a organization to be able to run it locally. This command below can help you out:
```

Comment:
Maybe we can add a note about the possibility of using our management views in the organization settings to create the app definition.